{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [],
        "fix_commit": "https://github.com/python-social-auth/social-app-django/commit/31c3e0c7edb187004d8abbde7e9c4f7ef9098138",
        "version_end_excluding": {},
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "BSD 3-Clause \"New\" or \"Revised\" License"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2024-32879"
        ],
        "CWE": [
            "CWE-178",
            "CWE-303"
        ]
    }
}